mungegithub/mungers/flake-manager.go

```diff
@@ -79,7 +79,7 @@ func (p *FlakeManager) Initialize(config *github.Config, features *features.Feat
 	p.config = config
 	p.googleGCSBucketUtils = utils.NewUtils(utils.KubekinsBucket, utils.LogDir)
 
-	var owner *testowner.ReloadingOwnerList
```

Comment:
Ahh, so the implicit conversion from *testowner.ReloadingOwnerList to sync.OwnerMapper creates a non-nil interface with type pointing at *testowner.ReloadingOwnerList and the data pointing at nil.